package.json

```diff
@@ -9,10 +9,10 @@
     "server": "hexo server"
   },
   "hexo": {
-    "version": "4.0.0"
```

Comment:
Please don't change the hexo version as it causing the build to fail. Revert the changes done in package.json